src/xdocs/config_filters.xml

```diff
@@ -417,7 +417,7 @@ public class UserService {
           <p>
             Filter <code>SuppressionFilter</code> rejects
             audit events for Check errors according to
-            a <a href="config.html#XML_Structure"><em>suppressions XML
+            a <a href="dtds/suppressions_1_2.dtd"><em>suppressions XML
```

Comment:
I do not see a reason why to keep it. DTD of this filter should be explained in it own documentation section.
Please explain why you want to keep link to XML_Structure .

Reply:
This was because the section explained the DTD. I am fine to keep documentation for it here since we removed the old one, and with new section, discussed below, we could just link to that instead.

Reply:
new subsection will be a links to examples, it is another issue - #5395. Ones we agree on format, we can make it "approved".
new subsection will not explain DTD, I try to avoid DTD explanation in xdoc and let user read it himself, but ideally just looks at examples and make what he need.